classes/Exception/ThankableNotFound.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Claromentis\ThankYou\Exception;
+
+class ThankableNotFound extends ThankableException
```

Comment:
`ThankableNotFoundException`. We suffix all exception classes with `Exception`.

Reply:
Fixed in 0d29269094fc75b1ddd3a0166743f2803d2d55a8